server/smc/smccli/controller/controller.go

```diff
@@ -0,0 +1,65 @@
+package controller
+
+import (
+	"go.dedis.ch/dela/cli"
+	"go.dedis.ch/dela/cli/node"
+	"go.dedis.ch/kyber/v3/suites"
+	"go.dedis.ch/kyber/v3/util/key"
+)
+
+// smcctl is an initializer with a set of commands for the SMC. It only
+// creates and injects a new SMC instance.
+//
+// - implements node.Initializer
+type smcctl struct {
+	kp *key.Pair
+}
+
+// NewSmcController returns a new SMC initializer
+func NewSmcController() node.Initializer {
+	return smcctl{
+		kp: key.NewKeyPair(suites.MustFind("Ed25519")),
+	}
+}
+
+// Build implements node.Initializer. In this case we don't need any command.
+func (s smcctl) SetCommands(builder node.Builder) {
+	cmd := builder.SetCommand("smc")
+	cmd.SetDescription("SMC service administration")
+
+	sub := cmd.SetSubCommand("createkeys")
+	sub.SetDescription("create key pair for reencryption")
+	sub.SetAction(builder.MakeAction(createKpAction{}))
```

Comment:
as we discussed, this should be a local action. No need for `builder.MakeAction`, you can just use pure functions.

Reply:
done